test/peer-store/key-book.spec.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+/* eslint-env mocha */
+
+const chai = require('chai')
+chai.use(require('dirty-chai'))
+chai.use(require('chai-bytes'))
+const { expect } = chai
+const sinon = require('sinon')
+
+const PeerStore = require('../../src/peer-store')
+
+const peerUtils = require('../utils/creators/peer')
+const {
+  codes: { ERR_INVALID_PARAMETERS }
+} = require('../../src/errors')
+
+describe('keyBook', () => {
+  let peerId, peerStore, kb
+
+  beforeEach(async () => {
+    [peerId] = await peerUtils.createPeerId()
+    peerStore = new PeerStore()
+    kb = peerStore.keyBook
+  })
+
+  it('throwns invalid parameters error if invalid PeerId is provided in set', () => {
+    try {
+      kb.set('invalid peerId')
+    } catch (err) {
+      expect(err.code).to.equal(ERR_INVALID_PARAMETERS)
+      return
+    }
+    throw new Error('invalid peerId should throw error')
```

Comment:
No need to change but you could instead do something like:
```js
expect(() => kb.set('invalid peerId')).to.throw()
      .that.has.property('code', ERR_INVALID_PARAMETERS)
```
which avoids potentially forgetting the return/throw statements when a failure occurs.

Reply:
I tried this out and I got:

```
AssertionError: expected [Function] to have property 'code'
```

I will skip this for now, we can revisit later